src/cljs/rems/user.cljs

```diff
@@ -53,22 +56,33 @@
    (example "empty attributes"
             [username {}])
    (component-info attributes)
-   (example "full set of attributes"
+   (example "full set of attributes, false invited-user status"
+            [attributes {:userid "developer@uu.id"
+                         :email "developer@uu.id"
+                         :name "Deve Loper"
+                         :notification-email "notification@example.com"
+                         :organizations [{:organization/id "Testers"} {:organization/id "Users"}]
+                         :address "Testikatu 1, 00100 Helsinki"
+                         :researcher-status-by "so"
+                         :nickname "The Dev"}
+             false])
+   (example "full set of attributes, true invited-user status"
             [attributes {:userid "developer@uu.id"
                          :email "developer@uu.id"
                          :name "Deve Loper"
                          :notification-email "notification@example.com"
                          :organizations [{:organization/id "Testers"} {:organization/id "Users"}]
                          :address "Testikatu 1, 00100 Helsinki"
                          :researcher-status-by "so"
-                         :nickname "The Dev"}])
-   (example "invalid value for researcher status"
+                         :nickname "The Dev"}
+             true])
```

Comment:
perhaps this case should only have `:email` and `:name` just like a realistic use case? not sure.

Reply:
Maybe, I probably can create another use case with "invited" user